### PR TITLE
Ignore configs

### PR DESCRIPTION
**Pull request by Abdur-rahmaanJ**
@Yahyakiani This is a review test, you get to review a PR!

### CODE REVIEW

File: shopyo/shopyoapi/cmd.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 from shopyoapi.file import trymkdir
 from shopyoapi.file import trymkfile
 from shopyoapi.file import get_folders
+from shopyoapi.file import trycopy
```
Comment (Yahyakiani) [inline shopyo/shopyoapi/cmd.py]: Import of 'trycopy' is not used. Remove it if it isn't necessary.

File: shopyo/app.py [hunk 2]
```diff
@@ -27,12 +26,27 @@
...
+    if not os.path.exists('config.py'):
+        trycopy('config_demo.py', 'config.py')
+    if not os.path.exists('config.json'):
+        trycopy('config_demo.json', 'config.json')
+except PermissionError as e:
+    raise e
+    print('Cannot continue, permission error'
+        'initialising config.py and config.json, '
+        'copy and rename them yourself!')
+    sys.exit()
```
Comment (Yahyakiani) [inline shopyo/app.py]: This Line is Unreachable. Kindly Fix That.

**Review by Yahyakiani — COMMENTED** [review]
Furthermore, Kindly Resolve the Pep8 errors as well.
1. All the Imports should be at the top of a file.
2. Remove any unnecessary line breaks.
3. Fix Indentation errors.
 To find line breaks simple use flake8 in your command terminal:
`flake8 path_of_file.py`
and it will tell you whether there are any issues with the file.
Thanks

File: shopyo/shopyoapi/cmd.py (hunk 1 above)
Comment (Abdur-rahmaanJ) [reply shopyo/shopyoapi/cmd.py]: Ok

**Comment by rehmanis** [comment]
@Yahyakiani if the changes have been made according to the review you provided then you can resolve them.

**Comment by Yahyakiani** [comment]
> 
> 
> @Yahyakiani if the changes have been made according to the review you provided then you can resolve them.

got it

**Comment by rehmanis** [comment]
@Yahyakiani not close the PR but resolve the review that you opened. Closing will cancel the PR.

image

**Comment by Yahyakiani** [comment]
> @Yahyakiani not close the PR but resolve the review that you opened. Closing will cancel the PR.
> 
> image
> 
 I have resolved the conversation s.
Since this is a test PR . Should I merge it ?

**Comment by rehmanis** [comment]
> > @Yahyakiani not close the PR but resolve the review that you opened. Closing will cancel the PR.
> > image
> 
> I have resolved the conversation s.
> Since this is a test PR . Should I merge it ?

I think merging should be fine. @Abdur-rahmaanJ can you confirm ?

**Comment by Abdur-rahmaanJ** [comment]
@Yahyakiani yes merge!

**Pull request merged**